docs/source/tutorials/token_classification/train.py

```diff
@@ -0,0 +1,793 @@
+from __future__ import absolute_import, division, print_function
+import json
+import os
+import torch
+import argparse
+import csv
+import logging
+import random
+import sys
+import torch.nn.functional as F
+from nltk import word_tokenize
+import numpy as np
+from pytorch_transformers import (WEIGHTS_NAME, AdamW, BertConfig,
+                                  BertForTokenClassification, BertTokenizer,
+                                  WarmupLinearSchedule)
+from torch import nn
+from torch.utils.data import (DataLoader, RandomSampler, SequentialSampler,
+                              TensorDataset)
+from torch.utils.data.distributed import DistributedSampler
+from tqdm import tqdm, trange
+from seqeval.metrics import classification_report
```

Comment:
are all these dependencies already installed with the ones specified in the training notebook? I don't think so, so we should specify these there as well as which versions of each package you used.